BuildAddOn.py

```diff
@@ -271,46 +274,61 @@ def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, pla
 
     elif platformName == 'MAC':
         subprocess.call ([
-            'cp', '-r',
+            'cp', '-R',
             sourceFolder / f'{addOnName}.bundle',
             packageFolder / f'{fileName}.bundle'
         ])
 
 
+def GetDevKitBuildNum (devKitData, version, platformName):
+    url = devKitData[platformName][version]
+    buildNum = url.split ('/')[-2]
+    return buildNum
+
+
 # Zip packages
-def PackageAddOns (args, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
+def PackageAddOns (args, devKitData, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
     Check7ZInstallation ()
 
     for version in acVersionList:
-        if args.release:
+        if args.allLocalizedVersions:
             for languageCode in languageList:
                 CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo', languageCode, True)

```

Comment:
Like in case of _BuildAddon_ calls inside _BuildAddons_ function, these shouldn't be called depending on the languages the addon was build with. If Package is called, we just want to build and upload RelWithDebInfo configs (with the pdb file included in case of WIN)

Reply:
Fixed